### ISSUE TITLE
Wrong IP is displayed for containers started outside the Rancher

### ISSUE THREAD

**Issue by y8** [rancher/rancher#2954]
If container is started before rancher-server or rancher-agent, the previous known IP address will be displayed in the Rancher UI, not the actual one.

Steps to reproduce:
1. Add 5 containers with `docker run --restart=always -d` on the host with rancher-server
2. Wait them to appear in the Rancher UI
3. Reboot the host

After reboot, in the Rancher UI, some containers will have wrong IP addresses.

Rancher: v0.47.0

Client:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:12:04 UTC 2015
 OS/Arch:      linux/amd64

Server:
 Version:      1.9.1
 API version:  1.21
 Go version:   go1.4.2
 Git commit:   a34a1d5
 Built:        Fri Nov 20 13:12:04 UTC 2015
 OS/Arch:      linux/amd64

**Comment by deniseschannon** [comment]
I am unable to reproduce. I followed the steps above and have rebooted my host a couple of times, even added new containers in between reboots. Every time the containers always display the correct IP. 

What OS are you using for the host? I was using an Ubuntu 14.04.3.

**Comment by y8** [comment]
`3.19.0-33-generic #38~14.04.1-Ubuntu SMP Fri Nov 6 18:17:28 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux`

For example:

![ntl](https://cloud.githubusercontent.com/assets/80930/11745758/c0f4aa6c-a01a-11e5-81b4-61c49ad96f5a.png)

**Comment by deniseschannon** [comment]
I was finally able to reproduce. I wonder if it's an order of operations as previously I was unable to reproduce. 

Steps To Reproduce:
1. Launch Rancher server without the `--restart-always` 
2. Add host using the machine that is also running rancher-server
3. Added 10 containers using the docker CLI on the same machine
 ` docker run --restart=always -d nginx`
4. Checked a couple of IPs of the containers that were deployed by doing a `ip add show eth0`. IPs all look good.
5.  Deleted 2 containers using the CLI on the machine.
6. Rebooted my machine. After reboot, I needed to start up rancher-server again.

The IPs of the 8 remaining containers are set differently, but the IPs in the API/UI are not updated.
